### PR TITLE
Create diary-study.md

### PR DESCRIPTION
**Pull request by AntoineGarciaSuarez**
here is an english draft. I clearly need your help for the translation ;)
Also I don't know any book in english about these subject.

### CODE REVIEW

File: _pages/en/methods/diary-study.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+layout: page
+title:  "Interviewing"
+lang: en
+category: "Methods"
+permalink: "/diary-study/"
+trans_url: "/journal-de-bord/"
+---
+
+### What is a diary study?
+Diary study are qualitative methods of collecting data about what people have done or felt when
```
Comment (marthaedwards) [inline _pages/en/methods/diary-study.md]: Minor grammar thing:
A diary study is a qualitative method for collecting data ....
OR 
Diary studies are a qualitative method for collecting data ...

File: _pages/en/methods/diary-study.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+title:  "Interviewing"
+lang: en
+category: "Methods"
+permalink: "/diary-study/"
+trans_url: "/journal-de-bord/"
+---
+
+### What is a diary study?
+Diary study are qualitative methods of collecting data about what people have done or felt when 
+interacting with a product or service. They allow to account for the experience over time, to evaluate it
```
Comment (marthaedwards) [inline _pages/en/methods/diary-study.md]: " They allow to account for the experience over time, to evaluate it 
and to study it in order to understand how a system is adopted and integrated into their everyday life."

I found this a bit confusing - maybe this:
"Diary studies account for the user experience over time, allowing researchers to evaluate, study and understand how a system is adopted and integrated into a person's everyday life."

File: _pages/en/methods/diary-study.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+trans_url: "/journal-de-bord/"
+---
+
+### What is a diary study?
+Diary study are qualitative methods of collecting data about what people have done or felt when 
+interacting with a product or service. They allow to account for the experience over time, to evaluate it 
+and to study it in order to understand how a system is adopted and integrated into their everyday life.
+Unlike field observations, the diary study does not require the presence of an observer (thus limiting observational or acquiescence bias). As a design researcher, we want to 
+capture the experience as close to the real thing as possible and not the one that is influenced by a controlled environment.
+The data collected help understanding the users, their activities or the context in which they are located.
```
Comment (marthaedwards) [inline _pages/en/methods/diary-study.md]: The data collected helps us understand the users .... 
(minor grammar)

File: _pages/en/methods/diary-study.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+### Key elements of a diary study
+### 1. Situations for using a diary study.
+Evaluation and exploration tools, diary studies are useful to:
```
Comment (marthaedwards) [inline _pages/en/methods/diary-study.md]: I would say take out "evaluation and exploration tools" and just say "diary studies are useful to: ... "

File: _pages/en/methods/diary-study.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+### 2. Manage a diary study.
+In terms of content, the diary study generally gathers factual data (date, time, place, activity) and subjective data (feelings, emotions, mood ...). Whether in paper or electronic format, this method requires motivated and conscientious participants who will have the rigor to enter their data frequently and regularly. 
+the participants will fill in the diary study according to the chosen protocol: 
+* at intervals
+* as soon as a signal asks them to do so
+* whenever a predefined event occurs
+Diary studies do not require a large number of participants. 
+To gather great insights, it is advisable to recruit at least 12 to 15 persons.
+
```
Comment (marthaedwards) [inline _pages/en/methods/diary-study.md:31]: This whole page is great!! really well done and helpful.

**Comment by cmaczo** [comment]
@AntoineGarciaSuarez Do you need help making the changes @marthaedwards suggests? Anything else we can do?

File: _pages/en/methods/diary-study.md (hunk 5 above)
Comment (AntoineGarciaSuarez) [reply _pages/en/methods/diary-study.md:31]: thanks!

**Pull request merged**